packages/ibm-products/src/components/Datagrid/Datagrid/DatagridToolbar.tsx

```diff
@@ -177,14 +190,22 @@ const DatagridBatchActionsToolbar = (
         toolbarBatchActions &&
         toolbarBatchActions?.map((batchAction, index) => {
           if (
-            (index < 2 && toolbarBatchActions.length > 3) ||
-            (index < 3 && toolbarBatchActions.length <= 3)
+            (!toolbarBatchActionsDisplayMin &&
+              index < 2 &&
+              toolbarBatchActions.length > 3) ||
+            (index < 3 && toolbarBatchActions.length <= 3) ||
+            (toolbarBatchActionsDisplayMin !== undefined &&
+              index < toolbarBatchActionsDisplayMin)
           ) {
             return (
               <TableBatchAction
                 key={`${batchAction.label}-${index}`}
                 renderIcon={batchAction.renderIcon}
                 onClick={(event) => onClickHandler(event, batchAction)}
+                className={cx(`${carbon.prefix}--noLabel`, {
```

Comment:
```suggestion
                className={cx({
```
Think you want something like this, since that class is now conditionally set